### ISSUE TITLE
Doesn't work on iOS.

### ISSUE THREAD

**Issue by KonstantinZhukovskij** [killserver/react-native-screenshot-prevent#1]
**Doesn't work on iOS platforms.** I can take screenshots, the background blurs the next second. As a result, I get a full screenshot of the application.

react-native v.0.67.1.

**Comment by killserver** [comment]
yes. I am currently working on a blocking method for iOS. I think within a month-two i will be possible to fix this problem

**Comment by ZaidQ797** [comment]
guys you can't prevent to take screenshots on ios due to OS limitations but you can block to record screen

**Comment by crzycoder** [comment]
@killserver any update on IOS prevention.. Still take screenshot in IOS

**Comment by feraswfares** [comment]
@crzycoder 

did you find any solution to prevent take screenshot in IOS

**Comment by crzycoder** [comment]
No I can't

**Comment by feraswfares** [comment]
@crzycoder 

thank you for response 

actually i will try the first method 
You can't prevent screen capture on iOS.

I have suggested 2 methods

1) After the user capture screen (UIApplicationUserDidTakeScreenshot), send the capture event to the server or display a warning message.
2) Try ScreenShieldKit (https://screenshieldkit.com/). You can read more from this article. https://medium.com/nomtek/screenshot-preventing-on-mobile-apps-9e62f51643e9

**Comment by crzycoder** [comment]
Shield kit is paid service

**Comment by feraswfares** [comment]
@crzycoder 

i know i will try to use this UIApplicationUserDidTakeScreenshot to detect when the user use screenshot i will display a warning message when he try it again i will block his account
